.github/workflows/build-metrics-server.yml

```
@@ -0,0 +1,110 @@
+jobs:
+  build-and-push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: prepor/action-aws-iam-authenticator@master
+      - run: aws-iam-authenticator version
+
+      - name: Install Kubectl
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:7: Double quote to prevent globbing and word splitting

(shellcheck)